Anim system from VScript

Shadowysn opened the pull request:
Ported animation system from VScript version.
Fixed plugin-reload bug that results in completely-static leftover legs that persist even when plugin is rerun.
NOTE: People on my modpage reported a very inconsistent bug where the pose parameter that moves the arms seems to fail. I've never been able to recreate this bug for myself, and I can only guess it's a client-side issue and might be because of me setting the parameter to **0.0** instead. _(this fork request has 0.0 as well)_
Is this why you constantly reset the pose parameter to **-40.0** despite the netprop actually operating between **0.0** _(lowest pose)_ to **1.0** _(highest pose)_? I ask as the former method causes horrible console spam in developer mode.

Shadowysn committed:
Anim system from VScript

Ported animation system from VScript version.
Fixed plugin-reload bug that results in completely-static leftover legs that persist even when plugin is rerun.

LuxLuma commented:
Thanks for the pull ill review it, in few weeks.

The issues i'v only seen with the arms was due to clientside model mods, i'v not saw this kind of issue with the default ones.

The netprop value of -40.0 was just arbitrary I had no idea how it worked tbh at the time.

Shadowysn committed:
Update _[L4D2]Survivor_Legs.sp

Updated to detect 1 HP forced walking limp and actually properly detect temporary health.

LuxLuma reviewed (approved):
Tested for awhile now, no issues at all thanks ill go ahead and merge these changes.
Thanks
